app/src/components/containers/AttributeListFormField/attributeListFormField.jsx

```diff
@@ -66,11 +66,7 @@ export const AttributeListFormField = ({
 
   return (
     <>
-      <Checkbox
-        value={shown}
-        onChange={attributeControlHandler}
-        data-automation-id={'showAttributesCheckbox'}
```

Comment:
Is it no longer needed for UI automation purposes?

Reply:
From the task:
"Also, please remove 'data-automation-id="showAttributesCheckbox"', its unnecessary now"